SDK/AppCenter/Microsoft.AppCenter.WindowsDesktop/Utils/DeviceInformationHelper.cs

```diff
@@ -81,6 +81,11 @@ protected override string GetDeviceModel()
                 AppCenterLog.Warn(AppCenterLog.LogTag, "Failed to get device model. Make sure that WMI service is enabled.", exception);
                 return string.Empty;
             }
+            catch (PlatformNotSupportedException exception)
+            {
+                AppCenterLog.Warn(AppCenterLog.LogTag, "Failed to get device model. Make sure that .NET Framework is up to date.", exception);
```

Comment:
This message can be confusing. Does the exception occur on a specific versions of .NET Framework? We need to specify the minimum supported version in that case.
Same question for the similar logs below.

Reply:
I'm afraid I can't answer this question, I fiddled a bit with ILSpy in the .NET DLL throwing the exception and it seems to just throw if any of the needed symbols are not found in the Win32 DLL, without providing any further information...

Reply:
So the idea is that even the `System.Management` package coming from NuGet tries to load `C:\Windows\Microsoft.NET\Framework64\v4.0.30319\WMINet_Utils.dll` and this DLL comes with Windows.

Reply:
Got it, thanks.